docs/image-fr.md

```diff
@@ -0,0 +1,39 @@
+# Utiliser une image préconfigurée
```

Comment:
Could you change the filename to make it consistent?
```diff
- docs/image-fr.md
+ docs/image.fr.md
```

Reply:
I changed the filename, however i'm not sure if Github got it right :/